DataConnectors/Trend Micro/AzureFunctionTrendMicroXDR/TimerTrigger/__init__.py

```diff
@@ -0,0 +1,143 @@
+# Tiite: Trend Micro XDR Data Connector
+# Language: Python 3.8
+# Author: Trend Micro 
+# Last Modified: 11/11/2020
+# Comment: Intial Release
+
+import datetime
+import logging
+import requests
+import base64
+import json
+import datetime
+import hmac
+import hashlib
+import sys
+import os
+import azure.functions as func
+
+def main(mytimer: func.TimerRequest) -> None:
+    utc_timestamp = datetime.datetime.utcnow().replace(
+        tzinfo=datetime.timezone.utc).isoformat()
+
+    if mytimer.past_due:
+        logging.info('The timer is past due!')
+        
+
+    logging.info('Python timer trigger function ran at %s', utc_timestamp)
+
+#Global Variables
+
+region = {'us': 'https://api.xdr.trendmicro.com', 'eu': 'https://api.eu.xdr.trendmicro.com', 'in': 'https://api.in.xdr.trendmicro.com',
+          'jp': 'https://api.xdr.trendmicro.co.jp', 'sg': 'https://api.sg.xdr.trendmicro.com', 'au': 'https://api.au.xdr.trendmicro.com'}
+
+customer_id = os.environ['workspaceId'] 
+shared_key = os.environ['workspaceKey']
+api_id = os.environ ['api_key']
+regioncode = os.environ ['regioncode']
+url_base = region[regioncode]
+log_type = 'TrendMicro_XDR'
+
+#Get List of Events
+def getWorkbenchList():
+    query_params = 'source=all&investigationStatus=null&sortBy=createdTime&queryTimeField=createdTime&offset=0&limit=100'
+    headers = {'Authorization': 'Bearer ' + api_id, 'Content-Type': 'application/json;charset=utf-8'}
+    url = url_base + '/v2.0/siem/events' + '?' + query_params
+    response = requests.get(url, headers=headers)
+    workbench_list = response.json()
+    return workbench_list
+
+# Get each Workbench Data from API and return Json Data
+def getWorkbench(workbencheIds):
+    url_path1 = '/v2.0/xdr/workbench/workbenches/'+workbencheIds

```

Comment:
Not sure why you are calling these `url1` and `url_path1`, the variables are scoped to the method so should be fine to use `url`/`url_path`